app/src/main/java/app/grapheneos/camera/capturer/ImageSaver.kt

```diff
@@ -84,6 +89,7 @@ class ImageSaver(
         }
 
         imageWriterExecutor.execute(this::saveImage)
+        Log.i("TAG", "Trying to save image $photosInQueue")
```

Comment:
Don't leave verbose logging in release builds.
Use `if (BuildConfig.DEBUG)` or similar check.
Also, reference `TAG` constant, not `"TAG"` string.
Same applies to line 132.

Reply:
Hey @muhomorr,

Actually those log checks were only present to make debugging the app a bit easier while testing/reviewing. I'll just remove these logs soon. Thanks for mentioning about them @muhomorr!